pkg/volume/csi/csi_attacher_test.go

```diff
@@ -1023,23 +1024,32 @@ func TestAttacherGetDeviceMountPath(t *testing.T) {
 	csiAttacher := getCsiAttacherFromVolumeAttacher(attacher, testWatchTimeout)
 
 	pluginDir := csiAttacher.plugin.host.GetPluginDir(plug.GetPluginName())
-
 	testCases := []struct {
-		testName          string
-		pvName            string
-		expectedMountPath string
-		shouldFail        bool
+		testName      string
+		pvName        string
+		volumeId      string
+		skipCSISource bool
+		shouldFail    bool
+		addVolSource  bool
```

Comment:
oh, I missed it. Could you add comments about this option?

Reply:
Done